dot/core/interface.go

```diff
@@ -71,6 +71,7 @@ type BlockState interface {
 	GetBlockBody(hash common.Hash) (*types.Body, error)
 	HandleRuntimeChanges(newState *rtstorage.TrieState, in runtime.Instance, bHash common.Hash) error
 	GetRuntime(*common.Hash) (runtime.Instance, error)
+	GetRuntimeFromDB(hash *common.Hash) (runtime.Instance, error)
```

Comment:
Didn't we want to not have this method, and instead call a method on storagestate in callers?

Reply:
I'm not sure I understand what you mean here?  Should GetRuntimeFromDB be called from GetRuntime if that returns nil?

Reply:
No we should get the runtime from `StorageState`'s `LoadCode` instead of from `blockState` I think.